src/App.js

```diff
@@ -1,27 +1,10 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
```

Comment:
- [ ] Great job so far @kambereBr  You have done an impressive job :+1: There are just a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Per the project requirements, Kindly ensure that all the boilerplate that got created by the CRA (texts, images, and styles) is deleted. Since you are making use of `App.js` and `index.js`, those aren't inclusive :+1: 
- Your current file structure
![image](https://github.com/noelfoka/math-magicians/assets/102699958/79272ce4-1231-4288-a5bf-ebd8fe6097fe)

Reply:
@NahnahAJ thanks for your review. I've cleaned the project.